k8s/deployments/kes-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kes
+  labels:
+    name: kes
+spec:
+  ports:
+    - port: 7373
+  selector:
+    app: kes
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kes
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kes
+  template:
+    metadata:
+      labels:
+        app: kes
+    spec:
+      containers:
+        - name: kes
+          image: minio/kes:latest
+          imagePullPolicy: "IfNotPresent"
+          command: ["kes"]
+          args: ["server", "--config=kes-config/server-config.toml", "--mtls-auth=ignore"]
+          ports:
+            - containerPort: 7373
+          volumeMounts:
+            - name: configuration
+              mountPath: "/kes-config"
+              readOnly: true
+            - name: server-keypair
+              mountPath: "/kes-config/server"
+              readOnly: true
+#            - name: app-keypair
+#              mountPath: "/kes-config/app"
+#              readOnly: true
+      volumes:
+        - name: configuration
+          secret:
+            secretName: acme-kes-server-config
+        - name: server-keypair
+          secret:
+            secretName: acme-kes-server-keypair
+#        - name: app-keypair
+#          secret:
+#            secretName: acme-kes-app-keypair
```

Comment:
are this needed? or can you add a comment for their purpose in case of uncommented

Reply:
This was for debugging purposes, I'll remove the file